eo.txt

```diff
@@ -2567,7 +2568,7 @@ client_packet(Guild, Create)
 {
 	int session_id
 	break
-	raw_string(3) guild_tag
```

Comment:
I'm not sure about these though.. `string` implies there's always a break after. Need to do some testing with GameServer to confirm

Reply:
Confirmed there is a `break` after tag